Profiling_task_memory output.txt

```diff
@@ -0,0 +1,125 @@
+C:\ProgramData\Anaconda3\python.exe "C:/Users/Алексей/Desktop/EPAM/EPAM Python stage 2/Profiling/Profiling_task_memory.py"

```

Comment:
Please provide an analysis of your results. Try to explain why we got these results. Which implementation is best and why?